testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/suspend/SuspendOnSoftKillTestCase.java

```diff
@@ -210,18 +219,6 @@ public Object call() throws Exception {
                 // ignore; it's fine if the server's down
             }
 
-            final HttpURLConnection conn = (HttpURLConnection) new URL(address).openConnection();
```

Comment:
Should we keep this call and just check it is a 5xx response (as it can be either 500 or 503 aiui)?

Reply:
It can actually be 500, 503 or a complete failure (IOException), so I don't know if it i really proves much.